docs/getting-started/tutorial-sample-data.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[tutorial-sample-data]]
+== Exploring {kib} using sample data
+
+You’re new to {kib} and want to try it out, but don’t have any data. No worries. 
+The Flights sample data set can help you get comfortable visualizing {es} data.
+
+If security is enabled, you must have the `all` Kibana privilege.
+You must also have access to the `.kibana_sample_data_flights` index with 
+the `read`, `write,` and `manage` privileges. See {xpack-ref}/security-privileges.html[Security Privileges] 
+for more information.
+
+The Flights data set analyzes data for five airlines.  It’s available from the {kib} home page. 
+
```

Comment:
The data set doesn't analyze. :-) I'd probably say that "You can load the data and preconfigured dashboard from the home page" instead of "It's available..."

Reply:
Also, there are four airlines. One for each core component of the stack 😄